packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -204,6 +218,38 @@ export const graphGridlineColor = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.graphGridlineColor)
 ).withDefault((element: HTMLElement) => hexToRgbaCssColor(getColorForTheme(element, Black91, Black15, White), 0.2));
 
+export const graphTrace1Color = DesignToken.create<string>(
```

Comment:
Was not described in the issue, comments, and no prior discussion so guess we need to discuss in owner's PR (which I'd prefer not to do @m-akinc)?

I'd prefer the color theme to not be just completely broken. Using White for all the traces is just completely unusable. Maybe we should just pick either the light or dark colors as a temporary solution? And then we should capture usage guidance somewhere that says Color isn't fully supported yet. Thoughts @jattasNI @m-akinc

Reply:
> Was not described in the issue, comments, and no prior discussion so guess we need to discuss in owner's PR (which I'd prefer not to do @m-akinc)?

I didn't think there was a need for discussion. Per the design, we cannot provide graph trace tokens that work generally for monochromatic applications:
![image](https://github.com/ni/nimble/assets/7282195/5b6fe313-70d2-4ff4-a05d-af4db4453af1)
If the use case only requires a single trace, then it seems correct for that trace (`graphTrace1Color`) to be white for the color theme. If there are more traces, then based on what Rick wrote in his design, the client presumably has to override the token values based on the background color. So defaulting all the traces to white seems reasonable. I'll add this rationale to the PR description.

Reply:
Our general idea is that even for unsupported configurations we don't completely fail and we give clear usage guidance. This is a complete failure where if we choose any color palette a user might be able to partially use the control in a poorly configured app this is just completely unusable.

> So defaulting all the traces to white seems reasonable.

I disagree with that assessment. 

>>  And then we should capture usage guidance somewhere that says Color isn't fully supported yet.
> I'll add this rationale to the PR description.

That concern is still not addressed.

Reply:
Another option: Use 8 different shades of white with trace 1 being full white. That way the I guess expected trace 1 color of white is still full white and there is some differentiation between traces 2-8 even if not great. And then documenting usage guidance.

Maybe we should have a Plot mdx that describes what tokens we have available for styling a plot and can also capture usage guidance.

Reply:
Ok, I changed the color theme defaults to various opacities of white (using grayscale shades at full opacity looked odd). I also added a "Graph Styling" mdx that lists the graph-related tokens and gives usage guidance about overriding trace colors for the color theme.